pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+[//]: # (You should also always follow the Pull Request Template when opening a PR. Following the pull request template is an important step to ensure that the information is presented in a clean and readable way.)
+[//]: # (Before a Pull Request is merged it will go through a couple of checks to makes sure the fix works and is compatible.)
+[//]: # (The PR will be built by the CI's.)
```

Comment:
(All Pull Requests are built by Travis CI & Appveyor to ensure a successful build.)